### PR TITLE
Update Chinese localization

### PR DESCRIPTION
**Pull request by KuwaLee**
TW and CN are included, co-maintain by @auroursa @cirx1e

### CODE REVIEW

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -857,11 +857,11 @@ msgstr "點擊這裡"
 
 #: src/view/com/modals/DeleteAccount.tsx:208
 msgid "Click here for more information on deactivating your account"
-msgstr ""
+msgstr "點擊這裡以了解有關停用帳號的更多信息"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:860]:
```suggestion
msgstr "點擊這裡以瞭解有關停用帳號的詳細資訊"
```

File: src/locale/locales/zh-TW/messages.po [hunk 2]
```diff
@@ -5044,7 +5044,7 @@ msgstr "服務條款已遷移到"
 
 #: src/screens/Settings/components/DeactivateAccountDialog.tsx:86
 msgid "There is no time limit for account deactivation, come back any time."
-msgstr ""
+msgstr "帳號停用沒有時間限制，隨時都可以充新啟用。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5047]:
```suggestion
msgstr "帳號停用沒有時間限制，隨時都可以重新啟用。"
```

錯字

File: src/locale/locales/zh-TW/messages.po [hunk 3]
```diff
@@ -5988,7 +5988,7 @@ msgstr "您現在可以使用新密碼登入。"
 
 #: src/screens/Deactivated.tsx:136
 msgid "You can reactivate your account to continue logging in. Your profile and posts will be visible to other users."
-msgstr ""
+msgstr "您可以登入以重新啟用帳號。其他用戶可以看到您的個人檔案和貼文。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:5991]:
```suggestion
msgstr "您可以登入以重新啟用帳號。其他用戶將可以重新看到您的個人檔案和貼文。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 4]
```diff
@@ -6130,7 +6130,7 @@ msgstr "輪到您了"
 #: src/screens/Deactivated.tsx:89
 #: src/screens/Settings/components/DeactivateAccountDialog.tsx:54
 msgid "You're logged in with an App Password. Please log in with your main password to continue deactivating your account."
-msgstr ""
+msgstr "您使用應用程式專用密碼登入。請使用您的主密碼登錄，以繼續停用您的帳號。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:6133]:
```suggestion
msgstr "您正在使用應用程式專用密碼登入。請使用您的主密碼登錄，以繼續停用您的帳號。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 5]
```diff
@@ -857,11 +857,11 @@ msgstr "點擊這裡"
 
 #: src/view/com/modals/DeleteAccount.tsx:208
 msgid "Click here for more information on deactivating your account"
-msgstr ""
+msgstr "點擊這裡以了解有關停用帳號的更多信息"
 
 #: src/view/com/modals/DeleteAccount.tsx:216
 msgid "Click here for more information."
-msgstr ""
+msgstr "點擊這裡以了解更多信息。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:864]:
```suggestion
msgstr "點擊這裡以瞭解更多資訊。"
```

**Comment by pfrazee** [comment]
Is this good to go?

**Comment by KuwaLee** [comment]
@pfrazee It's ready to go!!

**Review by pfrazee — APPROVED** [review]
Thanks!!

**Pull request merged**